sass/main.scss

```diff
@@ -883,8 +883,14 @@ nav {
     }
 
     .events {
+        $hover-translate-magnitude: 4px;
+
         width: 60%;
 
+        // Prevent event cards from clipping during their hover animation
```

Comment:
Without this, the top of the first card clips during the animation due to `overflow: hidden` on some of the `.events` elements.